net.gaijin.WarThunder.yaml

```diff
@@ -0,0 +1,31 @@
+app-id: net.gaijin.WarThunder
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+command: run.sh
```

Comment:
I think you are confusing the script to run the application and the script to apply the extra data.

Also I don't think you can use the extra data for the source of the icons. Definitely `/app` is read only.

Reply:
`/app` is only read-only during runtime. It should work during install time and build time.

Reply:
what I'm getting it is that run.sh contain things that belong to the `apply_extra`.

There should be `/app/bin/apply_extra` that will extract the extra_data. and `run.sh` listed above is the the script that whill launch the app (which in that case has `/app` in read only)

Reply:
Added apply_extra to extract needed files during install